### PR TITLE
Update first.txt

### PR DESCRIPTION
**Pull request by manishaotari**
updated file

### CODE REVIEW

**Comment by manishaotari** [comment]
[{"id":"9303fccb.3e35b","type":"tab","label":"Flow 5","disabled":false,"info":""},{"id":"3abc416c.6537de","type":"visual-recognition-v3","z":"9303fccb.3e35b","name":"Identify image","vr-service-endpoint":"https://gateway.watsonplatform.net/visual-recognition/api","image-feature":"classifyImage","lang":"en","x":396,"y":180,"wires":[["35d3819.5ccd47e"]]},{"id":"35d16b8e.d9da04","type":"function","z":"9303fccb.3e35b","name":"","func":"msg.params={\"classifier_ids\":\"FashionProductClassifier_1711266320\"}\n\nreturn msg;","outputs":1,"noerr":0,"initialize":"","finalize":"","x":221,"y":44,"wires":[[]]},{"id":"35d3819.5ccd47e","type":"function","z":"9303fccb.3e35b","name":"","func":"msg.payload=msg.result.images[0].classifiers[0].classes\nreturn msg;","outputs":1,"noerr":0,"initialize":"","finalize":"","x":332,"y":289,"wires":[["7e56519d.a77f3","b399db0a.778dd8"]]},{"id":"b399db0a.778dd8","type":"ui_template","z":"9303fccb.3e35b","group":"47197acb.fa2454","name":"","order":1,"width":"25","height":"11","format":"<!DOCTYPE html>\n<html lang=\"en\">\n<head>\n  <title>Visual Recognition</title>\n  <meta charset=\"utf-8\">\n  <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">\n   <link rel=\"stylesheet\" href=\"https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css\">\n  <script src=\"https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js\"></script>\n  <script src=\"https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js\"></script>\n  <script src=\"https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js\"></script>\n  <style>\n  .bg-light {\n    background-color: #00838F!important;\n\t}\n\th4 {\n    color: #fff;\n\t}\n\t\n\t.custom-file {\n\t\tmargin-bottom: 14px;\n\t\t}\n\t\t\n\t.table .thead-dark th {\n\t\tcolor: #fff;\n\t\tbackground-color: #307d76;\n\t\tborder-color: #307d76;\n\t}\n\th5{\n\t    text-align: center;\n\t\t    color: #bd6666;\n\t\t}\n\t\t.text-center {\n\t\t  text-align: center;\n\t\t}\n\t.imgdiv\n\t{\n\t\talign:center;\n\t}\n\t</style>\n</head>\n<body>\n\t<nav class=\"navbar navbar-expand-sm bg-light\">\n\t\t<div class=\"justify-content-center\">\n\t\t\t<h4 class=\"text-center\">Object Detection using Visual Recognition </h4>\n\t\t</div>\n\t</nav>\n\t<br><br>\n    <div class=\"container\">\n\t\t<div class=\"row\">\n\t\t\t<div class=\"col-sm-2\">\n\t\t\t</div>\n\t\t\t<div class=\"col-sm-8\">\n\t\t\t\t\n\t\t\t\t\t<div class=\"custom-file\">\n\t\t\t\t\t\n\t\t\t\t\t\t<input type=\"file\" name=\"pic\" accept=\"image/*\" onchange=\"readURL(this);\" class=\"custom-file-input\" id=\"customFile\">\n\t\t\t\t\t\t\n\t\t\t\t\t\t<label class=\"custom-file-label\" for=\"customFile\">Choose file</label>\n\t\t\t\t\t</div>\n\t\t\t\t\t<br>\n\t\t\t\t\t<div class=\"imgdiv\">\n\t\t\t\t\t\t\n\t\t\t\t\t</div>\n\t\t\t\t\t\n\t\t\t\t\t<button type=\"submit\" ng-click=\"send({payload:action()})\" class=\"btn btn-success\">Submit</button>\n\t\t\t\t\n\t\t\t\t\n\t\t\t</div>\n\t\t\t<div class=\"col-sm-2\">\n\t\t\t</div>\n\t\t</div>\n\t\t<div class=\"row\">\n\t\t\t<div class=\"col-sm-2\">\n\t\t\t</div>\n\t\t\t<div class=\"col-sm-8\">\n\t\t\t<h5> The Recognised Objects  </h5>\n\t\t\t<table class=\"table\">\n\t\t\t\t<thead class=\"thead-dark\">\n\t\t\t\t  <tr>\n\t\t\t\t\t<th>Class</th>\n\t\t\t\t\t<th>Score</th>\n\t\t\t\t  </tr>\n\t\t\t\t</thead>\n\t\t\t\t<tbody id=\"scoretable\">\n\t\t\t\t</tbody>\n\t\t\t  </table>\n\t\t\t  \n\t\t\t </div>\n\t\t\t<div class=\"col-sm-2\">\n\t\t\t</div>\n\t</div>\t\n</body>\n</html>\n<script>\nvar x=\"\";\n    function readURL(input) {\n        if (input.files && input.files[0]) {\n            var reader = new FileReader();\n\n            reader.onload = function (e) {\n                $('#blah')\n                    .attr('src', e.target.result)\n                    .width(150)\n                    .height(200);\n            };\n\n            reader.readAsDataURL(input.files[0]);\n            x= input.files[0]\n\n            \n            \n\n        }\n    }\n    function getdata(data)\n\t{\n\t\t var html = '';\n            if(data != 0)\n\t\t\t{\n\t\t\t\t$.each(data, function(i){\n\t\t\t\tvar row = data[i];\n\t\t\t\tconsole.log(row);\n\t\t\t\thtml += '<tr>';\n\t\t\t\thtml += '<td>';\n\t\t\t\thtml +=  row.class;\n\t\t\t\thtml += '</td>';\n\t\t\t\thtml += '<td>';\n\t\t\t\thtml +=  row.score;\n\t\t\t\thtml += '</td>';\n\t\t\t\thtml += '</tr>';\n\t\t\t});\n\t\t\t}\n\t\t\telse\n\t\t\t\thtml+=\"<div>No Data</div>\";\n\t\t\t$('#scoretable').html(html);\n\t}\n    (function(scope) {\n        scope.$watch('msg.payload', function(data) {\n            console.log('Position 2');\n            console.dir(data);\n            getdata(data);\n\n        });\n    })(scope);\n    \n    \n// or overwrite value in your callback function ...\nthis.scope.action = function() { return x; }\n\n</script>","storeOutMessages":true,"fwdInMessages":true,"resendOnRefresh":false,"templateScope":"local","x":145,"y":158,"wires":[["35d16b8e.d9da04","3abc416c.6537de"]]},{"id":"7e56519d.a77f3","type":"debug","z":"9303fccb.3e35b","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"payload","targetType":"msg","statusVal":"","statusType":"auto","x":711.5,"y":231,"wires":[]},{"id":"47197acb.fa2454","type":"ui_group","z":"","name":"Default","tab":"c86a8639.1869b8","order":1,"disp":false,"width":"27","collapse":false},{"id":"c86a8639.1869b8","type":"ui_tab","z":"","name":"Home","icon":"dashboard","disabled":false,"hidden":true}]
